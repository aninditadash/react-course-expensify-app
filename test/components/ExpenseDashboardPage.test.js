import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../src/components/ExpenseDashboardPage";

test("ExpenseDashboardPage Component: Should render ExpenseDashboardPage correctly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
