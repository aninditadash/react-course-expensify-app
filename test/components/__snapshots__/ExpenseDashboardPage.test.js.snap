// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboardPage Component: Should render ExpenseDashboardPage correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
